src/gtfs/read.rs

```diff
@@ -1405,9 +1405,7 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(
-        expected = "IdentifierAlreadyExists Error { id: \"1\", type: \"transit_model::objects::Network\" }"
-    )]
+    #[should_panic(expected = "`Err` value: identifier 1 already exists")]
```

Comment:
I'm not sure to understand this change. Any idea? This looks weird that before, it was a concrete error type (coming from `typed_index_collection` if I'm not wrong) and that now, it's probably converted into text because at some point, it's converted into `anyhow` (at least, this is my guess)? The same remark applies for the changes made in `tests/read_gtfs.rs` and `tests/read_ntfs.rs`.

Reply:
By the looks of it, anyhow errors have a special `Debug` implementation which is intended to be human readable (and very similar to `Display`) : https://github.com/dtolnay/anyhow/blob/871be23b2c3351157445c8f657917461ab33e6d7/src/fmt.rs#L19-L65.

On the other hand, it seems that `failure` was relying on the standard derive for `Debug`, which outputs the layout of the underlying struct.

Reply:
Thanks for the digging. Fine by me!